packages/jest-haste-map/src/crawlers/watchman.ts

```diff
@@ -125,6 +125,7 @@ export async function watchmanCrawl(options: CrawlerOptions): Promise<{
 
   const cmd = <T>(...args: Array<any>): Promise<T> =>
     new Promise((resolve, reject) =>
+      // @ts-expect-error: client is typed strictly, but incomplete
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71648

Very nice, but makes it harder to type when we wrap in promise (including node's `promisify`)